BrainPortal/app/views/tasks/show.html.erb

```diff
@@ -96,12 +96,17 @@
 
     <% if CbrainTask::FAILED_STATUS.include?(@task.status) or CbrainTask::COMPLETED_STATUS.include?(@task.status) %>
       <%= link_to "Remove Work Directory", url_for(:action  => :operation, :operation => 'zap_wd', 'tasklist[]' => @task.id),
-                                                  :method  => :post,
+                                                  :method   => :post,
                                                   :datatype => :script,
-                                                  :class => "button",
-                                                  :confirm =>
+                                                  :class    => "button",
+                                                  :confirm  =>
                   "Are you sure you want to remove the task's work directory?"
       %>
+      <%= link_to "Save Work Directory", url_for(:action    => :operation, :operation => 'save_wd', 'tasklist[]' => @task.id),
+                                                  :method   => :post,
+                                                  :datatype => :script,
```

Comment:
I don't think the `:datatype` is necessary at all ? Can it be removed form all these buttons?